### DISCUSSION TITLE
slow execution with gempy

### DISCUSSION THREAD

**Discussion by labriki** [gempy-project/gempy#859] [Q&A] [answered]
I use GemGis and Gempy tutorials to understand their uses and functionality, but I usually have a problem when I run **gp.compute_model(...)** nothing happens or it takes a long time to respond. how to avoid this?

**Comment by AlexanderJuestel** [comment]
Hello @labriki,

The runtime depends for instance on how many input data points you have and what the resolution is. A resolution of 50,50,50 cells is recommend. What are your settings and how many points do you have in your model?

**Reply by labriki** [reply]
Hi, 
In response to your questions and in example in tutorial AlesModel in Gempy, we have:
Points of Interface= 126 points;
Points of orientations = 198 points;
Resolution = [100, 100, 100]
I have this message on out: RuntimeWarning: overflow encountered in exp
  variables = ufunc(*ufunc_args, **ufunc_kwargs)

**Reply by AlexanderJuestel** [reply, marked as answer]
Try reducing the number of point and set the resolution to 50,50,50, this should reduce the computing time

**Reply by labriki** [reply]
Thinks, I'm going to try this. I think that in GenGis we must reduce the vertices when we digitalise features. isn't it?

**Reply by AlexanderJuestel** [reply]
As the input data is a GeoDataFrame, you can also just sample your gdf using `gdf=gdf.sample(n=your_number).reset_index(drop=True)`. This will reduce the number of points and accelerate the computing

**Reply by labriki** [reply]
Hi,
In example 18 of GemGis tutorial i used for example: 
gdf=interfaces
gdf=gdf.sample(n=100).reset_index(drop=True)
This input is after:  gg.vector.extract_xyz(gdf=interfaces, dem=topo_raster)
But i have this error message:
**ValueError**: Cannot take a larger sample than population when 'replace=False'.

**Reply by AlexanderJuestel** [reply]
This error usually appears if your n is larger than the actual number of interface points you have. Check maybe how many points you actually have

**Reply by labriki** [reply]
Hi and happy new year,
I'm sorry, the problem persists, even when I reduce the number of interface points
